pkg/controller/remoteca/remoteca_controller.go

```diff
@@ -0,0 +1,38 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package remoteca
+
+import (
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common/operator"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/remotecluster/remoteca"
+	"github.com/elastic/cloud-on-k8s/pkg/utils/rbac"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var (
+	name = "remoteca-controller"
+)
+
+// Add creates a new RemoteCa Controller and adds it to the manager with default RBAC.
+func Add(mgr manager.Manager, accessReviewer rbac.AccessReviewer, params operator.Parameters) error {
```

Comment:
Any particular reason why you did not move everything from `elasticsearch/remotecluster/remoteca` into this package?

Reply:
At first it was only in `elasticsearch/remotecluster/remoteca` The reason is that this controller only makes sense in the context of Elasticsearch.

I moved a part of it to make it "[visible](https://github.com/elastic/cloud-on-k8s/pull/2543#discussion_r382588005)", I'm not sure I would go further because this controller  is dependent on the Elasticsearch one, but I'm happy to move everything if you think it makes sense.